controllers/subscription.go

```diff
@@ -22,44 +22,45 @@ import (
 	"log"
 	"reflect"
 
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	olmclient "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	olmclient "github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
-
-	api "github.com/hybrid-cloud-patterns/patterns-operator/api/v1alpha1"
-	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"k8s.io/client-go/kubernetes"
 )
 
 const (
-	subscriptionNamespace = "openshift-operators"
-	applicationNamespace  = "openshift-gitops"
+	subscriptionNamespace  = "openshift-operators"
```

Comment:
Use the defaults you coded in defaults.go?

Reply:
Done.